pkg/api/pod/util.go

```diff
@@ -402,6 +404,17 @@ func haveSameExpandedDNSConfig(podSpec, oldPodSpec *api.PodSpec) bool {
 	return true
 }
 
+// haveInvalidTopologySpreadConstraintLabelSelector return true if constraints have any entry with invalid labelSelector
+func haveInvalidTopologySpreadConstraintLabelSelector(constraints []api.TopologySpreadConstraint) bool {
```

Comment:
nit: wonder if this should be named `hasInvalidTopologySpreadConstraintLabelSelector`?